.gitignore

```diff
@@ -56,4 +56,5 @@ typings/
 .yarn-integrity
 
 # dotenv environment variables file
-.env
\ No newline at end of file
+.env
+.env.example

```

Comment:
I'll remove the .env.example from .gitignore for now, but I don't think we should commit our .env files at all. 

Later on I suggest we rename .env.example to just .env though.